frontend/src/utils/idp.ts

```diff
@@ -86,6 +86,25 @@ export const identityProviderTemplateList: IdentityProviderTemplate[] = [
       },
     },
   },
+  {
```

Comment:
Thanks for the contribution. This standard idp list is only used for fully managed SaaS identity providers for better experience. For example, this list has GitLab (gitlab.com) instead of self-hosted GitLab. However, feel free to add a tutorial so that others can follow the instruction. Let @adela-bytebase know if you need any help. Thanks!

https://www.bytebase.com/tutorial